content/post/2021-09-24-r-ladies-feat-user-2021/index.en.md

```diff
@@ -0,0 +1,61 @@
+---
+title: R-Ladies+ Feat. useR! 2021
+author: ''
+date: '2021-09-24'
+slug: []
+categories:
+  - r-ladies
+  - R-Ladies
+tags:
+  - useR
+  - metameetup
+lastmod: '2021-09-24T14:59:29-03:00'
+keywords: []
+description: ''
+comment: no
+toc: no
+autoCollapseToc: no
+contentCopyright: no
+reward: no
+mathjax: no
+---
+
+![art_EN](https://user-images.githubusercontent.com/8397074/116813558-3e749e80-ab22-11eb-9c46-18fa17a3f1e7.png)
+
```

Comment:
In this post, you will see that we have used the R-Ladies logo with the colors of the rainbow and the trans flags. We have also added a + to the R-Ladies name.
As symbols and words matter a lot, we wanted to explicitly communicate, with the use of these symbols, the importance of diversity and inclusion to our community.  We used them in this useR! to reinforce that we welcome diversity and put an effort into creating collectively a safe and friendly space.